bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/handler/IntesisBoxHandler.java

```diff
@@ -203,32 +217,17 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
     }
 
-    private void receivedId(String function, String value) {
-        logger.trace("receivedID(): {} {}", function, value);
+    private void populateProperties(String data) {
+        String[] value = data.substring(3).split(",");
         properties.put(PROPERTY_VENDOR, "Intesis");
-        switch (function) {
-            case "MODEL":
-                properties.put(PROPERTY_MODEL_ID, value);
-                break;
-            case "MAC":
-                properties.put(PROPERTY_MAC_ADDRESS, value);
-                break;
-            case "IP":
-                properties.put("ipAddress", value);
-            case "PROTOCOL":
-                properties.put("protocol", value);
-                break;
-            case "VERSION":
-                properties.put(PROPERTY_FIRMWARE_VERSION, value);
-                break;
-            case "RSSI":
-                properties.put("rssi", value);
-                break;
-            case "NAME":
-                properties.put("hostname", value);
-                break;
-        }
-        logger.trace("Update Properties for ID : {}", properties);
+        properties.put(PROPERTY_MODEL_ID, value[0]);
+        properties.put(PROPERTY_MAC_ADDRESS, value[1]);
+        properties.put("ipAddress", value[2]);
+        properties.put("protocol", value[3]);
+        properties.put(PROPERTY_FIRMWARE_VERSION, value[4]);
+        properties.put("rssi", value[5]);
```

Comment:
Doesn't rssi change over time? If it changes over time it should be a channel, not a property

Reply:
Will change that later, as I have to think how to block other properties from regular updates. Should be no big issue.

Reply:
Done